### PR TITLE
Adding a note in the handlers section for people to better understand…

### PR DESCRIPTION
**Pull request by jamengual**
… how they work

### CODE REVIEW

File: source/docs/0.21/handlers.md [hunk 1]
```diff
@@ -65,6 +65,40 @@ The following is an example Sensu handler definition, a JSON configuration file
 }
 ~~~
 
+## Abrief note about handlers
+
+At startup the Sensu server reads all the json config files inside the `conf.d` directory and creates an object called `jsonconfig` that is basically a long json that includes all the config files read at starup separated by keys so if you have a file called `hipchat.json` whatever key name you define inside of it will be added to the `jsonconfig` object, for example :
+
+~~~json
+{
+  "hipchat-team1": {
```
Comment (hany) [inline source/docs/0.21/handlers.md:74]: Should this be simply `hipchat`? Your documentation below seems to suggest it should be `hipchat`?

File: source/docs/0.21/handlers.md [hunk 2]
```diff
@@ -65,6 +65,40 @@ The following is an example Sensu handler definition, a JSON configuration file
...
+    "apikey": "1234566gggg55555",
+    "apiversion": "v1",
+    "room": "Ops",
+    "from": "Sensu"
+  }
+}
+~~~
+
+The key `"hipchat"` will be added to the main sensu `jsonconfig` and will be use by the handler to find its config, so if you need to have multiple handlers with multiple configs you need to change the main key name and configure your handler properly to handle it. In this case for the hipchat handler the handler definition will look like this :
```
Comment (hany) [inline source/docs/0.21/handlers.md]: "and will be use by" - should be "used".

File: source/docs/0.21/handlers.md (hunk 1 above)
Comment (jamengual) [inline source/docs/0.21/handlers.md:74]: This is for a single handler with multiple configs, I think is clear but I can change it if you don't think so.

File: source/docs/0.21/handlers.md (hunk 2 above)
Comment (jamengual) [inline source/docs/0.21/handlers.md]: fixed.

File: source/docs/0.21/handlers.md (hunk 1 above)
Comment (hany) [inline source/docs/0.21/handlers.md:74]: The key "`hipchat`" is not referenced anywhere else except in the paragraph below this code block. I may be missing something though?

Comment (jamengual) [inline source/docs/0.21/handlers.md:74]: You are right, I think I fixed it now.

Comment (hany) [inline source/docs/0.21/handlers.md:74]: :+1:

**Comment by jamengual** [comment]
Is all good ?

**Comment by calebhailey** [comment]
Tagging this for 0.23 handler docs rewrite. Thanks for this contribution, @jamengual!

**Comment by calebhailey** [comment]
Closing this in favor of the major handler documentation rewrite in 0.23. See here for the latest: https://sensuapp.org/docs/latest/reference/handlers.html, and please feel free to submit revisions if you see room for further improvement.

**Pull request closed without merging**